2020-11-19 22:09:38.581 +01:00 [INF] User profile is available. Using 'C:\Users\Bartek\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-11-19 22:09:40.050 +01:00 [INF] Now listening on: https://localhost:5001
2020-11-19 22:09:40.057 +01:00 [INF] Application started. Press Ctrl+C to shut down.
2020-11-19 22:09:40.061 +01:00 [INF] Hosting environment: Development
2020-11-19 22:09:40.065 +01:00 [INF] Content root path: C:\Users\Bartek\source\repos\DP.Patients\DP.Patients
2020-11-19 22:09:48.808 +01:00 [INF] Request starting HTTP/1.1 POST https://localhost:5001/api/Patients application/json 0
2020-11-19 22:09:48.900 +01:00 [INF] Authorization failed.
2020-11-19 22:09:48.914 +01:00 [INF] AuthenticationScheme: Bearer was challenged.
2020-11-19 22:09:48.928 +01:00 [INF] Request finished in 137.2002ms 401 
2020-11-19 22:11:39.031 +01:00 [INF] Request starting HTTP/1.1 POST https://localhost:5001/api/Patients application/json; charset=utf-8 134
2020-11-19 22:11:39.749 +01:00 [INF] Successfully validated the token.
2020-11-19 22:11:39.754 +01:00 [INF] Authorization was successful.
2020-11-19 22:11:39.759 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients)'
2020-11-19 22:11:39.798 +01:00 [INF] Route matched with {action = "PostAdd", controller = "Patients"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] PostAdd(DP.Patients.Controllers.Model.Patient) on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:11:40.094 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients) - Validation state: "Valid"
2020-11-19 22:11:40.461 +01:00 [INF] Entity Framework Core 3.1.8 initialized 'DpDataContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2020-11-19 22:11:41.302 +01:00 [INF] Executed DbCommand (118ms) [Parameters=[@p0='?' (DbType = Int32), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (DbType = DateTime2)], CommandType='"Text"', CommandTimeout='30']
SET NOCOUNT ON;
INSERT INTO [Patients] ([Age], [Email], [Name], [Surname], [TestDate])
VALUES (@p0, @p1, @p2, @p3, @p4);
SELECT [Id]
FROM [Patients]
WHERE @@ROWCOUNT = 1 AND [Id] = scope_identity();
2020-11-19 22:11:42.206 +01:00 [INF] Executed action method DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients), returned result Microsoft.AspNetCore.Mvc.CreatedResult in 2103.0849ms.
2020-11-19 22:11:42.216 +01:00 [INF] Executing ObjectResult, writing value of type 'DP.Patients.Controllers.Model.Patient'.
2020-11-19 22:11:42.236 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients) in 2428.4032ms
2020-11-19 22:11:42.241 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients)'
2020-11-19 22:11:42.250 +01:00 [INF] Request finished in 3218.8643ms 201 application/json; charset=utf-8
2020-11-19 22:11:48.840 +01:00 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/Patients  
2020-11-19 22:11:48.845 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.GetData (DP.Patients)'
2020-11-19 22:11:48.850 +01:00 [INF] Route matched with {action = "GetData", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult GetData() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:11:48.875 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.GetData (DP.Patients) - Validation state: "Valid"
2020-11-19 22:11:48.905 +01:00 [INF] Entity Framework Core 3.1.8 initialized 'DpDataContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2020-11-19 22:11:49.127 +01:00 [INF] Executed DbCommand (80ms) [Parameters=[], CommandType='"Text"', CommandTimeout='30']
SELECT [p].[Id], [p].[Age], [p].[Email], [p].[Name], [p].[Surname], [p].[TestDate]
FROM [Patients] AS [p]
2020-11-19 22:11:49.150 +01:00 [INF] Executed action method DP.Patients.Controllers.PatientsController.GetData (DP.Patients), returned result Microsoft.AspNetCore.Mvc.OkObjectResult in 271.4908ms.
2020-11-19 22:11:49.154 +01:00 [INF] Executing ObjectResult, writing value of type 'System.Collections.Generic.List`1[[DP.Patients.Controllers.Model.Patient, DP.Patients, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]'.
2020-11-19 22:11:49.161 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.GetData (DP.Patients) in 306.4867ms
2020-11-19 22:11:49.164 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.GetData (DP.Patients)'
2020-11-19 22:11:49.168 +01:00 [INF] Request finished in 327.8612ms 200 application/json; charset=utf-8
2020-11-19 22:22:14.586 +01:00 [INF] User profile is available. Using 'C:\Users\Bartek\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-11-19 22:22:15.759 +01:00 [INF] Now listening on: https://localhost:5001
2020-11-19 22:22:15.765 +01:00 [INF] Application started. Press Ctrl+C to shut down.
2020-11-19 22:22:15.772 +01:00 [INF] Hosting environment: Development
2020-11-19 22:22:15.778 +01:00 [INF] Content root path: C:\Users\Bartek\source\repos\DP.Patients\DP.Patients
2020-11-19 22:22:16.270 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json; charset=utf-8 0
2020-11-19 22:22:17.136 +01:00 [INF] Successfully validated the token.
2020-11-19 22:22:17.147 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:22:17.179 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:22:17.485 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:22:21.375 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 4186.1357ms
2020-11-19 22:22:21.380 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:22:21.383 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:22:21.519 +01:00 [INF] Request finished in 5253.4743ms 500 text/plain
2020-11-19 22:22:28.396 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json; charset=utf-8 0
2020-11-19 22:22:28.402 +01:00 [INF] Successfully validated the token.
2020-11-19 22:22:28.405 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:22:28.410 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:22:28.452 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:22:30.503 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 2088.4866ms
2020-11-19 22:22:30.510 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:22:30.513 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:22:30.531 +01:00 [INF] Request finished in 2135.7673ms 500 text/plain
2020-11-19 22:30:28.140 +01:00 [INF] User profile is available. Using 'C:\Users\Bartek\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-11-19 22:30:29.115 +01:00 [INF] Now listening on: https://localhost:5001
2020-11-19 22:30:29.148 +01:00 [INF] Application started. Press Ctrl+C to shut down.
2020-11-19 22:30:29.151 +01:00 [INF] Hosting environment: Development
2020-11-19 22:30:29.156 +01:00 [INF] Content root path: C:\Users\Bartek\source\repos\DP.Patients\DP.Patients
2020-11-19 22:30:32.054 +01:00 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/Patients  
2020-11-19 22:30:32.136 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.GetData (DP.Patients)'
2020-11-19 22:30:32.193 +01:00 [INF] Route matched with {action = "GetData", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult GetData() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:30:32.448 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.GetData (DP.Patients) - Validation state: "Valid"
2020-11-19 22:30:32.826 +01:00 [INF] Entity Framework Core 3.1.8 initialized 'DpDataContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2020-11-19 22:30:33.653 +01:00 [INF] Executed DbCommand (98ms) [Parameters=[], CommandType='"Text"', CommandTimeout='30']
SELECT [p].[Id], [p].[Age], [p].[Email], [p].[Name], [p].[Surname], [p].[TestDate]
FROM [Patients] AS [p]
2020-11-19 22:30:33.766 +01:00 [INF] Executed action method DP.Patients.Controllers.PatientsController.GetData (DP.Patients), returned result Microsoft.AspNetCore.Mvc.OkObjectResult in 1308.1969ms.
2020-11-19 22:30:33.775 +01:00 [INF] Executing ObjectResult, writing value of type 'System.Collections.Generic.List`1[[DP.Patients.Controllers.Model.Patient, DP.Patients, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]'.
2020-11-19 22:30:33.825 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.GetData (DP.Patients) in 1621.4502ms
2020-11-19 22:30:33.829 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.GetData (DP.Patients)'
2020-11-19 22:30:33.836 +01:00 [INF] HTTP GET /api/Patients responded 200 in 1771.0941 ms
2020-11-19 22:30:33.846 +01:00 [INF] Request finished in 1796.955ms 200 application/json; charset=utf-8
2020-11-19 22:41:46.503 +01:00 [INF] User profile is available. Using 'C:\Users\Bartek\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-11-19 22:41:47.654 +01:00 [INF] Now listening on: https://localhost:5001
2020-11-19 22:41:47.658 +01:00 [INF] Application started. Press Ctrl+C to shut down.
2020-11-19 22:41:47.662 +01:00 [INF] Hosting environment: Development
2020-11-19 22:41:47.664 +01:00 [INF] Content root path: C:\Users\Bartek\source\repos\DP.Patients\DP.Patients
2020-11-19 22:41:55.153 +01:00 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/Patients  
2020-11-19 22:41:55.231 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.GetData (DP.Patients)'
2020-11-19 22:41:55.278 +01:00 [INF] Route matched with {action = "GetData", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult GetData() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:41:55.501 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.GetData (DP.Patients) - Validation state: "Valid"
2020-11-19 22:41:55.848 +01:00 [INF] Entity Framework Core 3.1.8 initialized 'DpDataContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2020-11-19 22:41:56.602 +01:00 [INF] Executed DbCommand (93ms) [Parameters=[], CommandType='"Text"', CommandTimeout='30']
SELECT [p].[Id], [p].[Age], [p].[Email], [p].[Name], [p].[Surname], [p].[TestDate]
FROM [Patients] AS [p]
2020-11-19 22:41:56.692 +01:00 [INF] Executed action method DP.Patients.Controllers.PatientsController.GetData (DP.Patients), returned result Microsoft.AspNetCore.Mvc.OkObjectResult in 1181.3533ms.
2020-11-19 22:41:56.699 +01:00 [INF] Executing ObjectResult, writing value of type 'System.Collections.Generic.List`1[[DP.Patients.Controllers.Model.Patient, DP.Patients, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]'.
2020-11-19 22:41:56.731 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.GetData (DP.Patients) in 1445.5209ms
2020-11-19 22:41:56.736 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.GetData (DP.Patients)'
2020-11-19 22:41:56.741 +01:00 [INF] HTTP GET /api/Patients responded 200 in 1577.3833 ms
2020-11-19 22:41:56.757 +01:00 [INF] Request finished in 1608.5808ms 200 application/json; charset=utf-8
2020-11-19 22:42:17.884 +01:00 [INF] Request starting HTTP/1.1 POST https://localhost:5001/api/Patients application/json; charset=utf-8 131
2020-11-19 22:42:18.551 +01:00 [INF] Successfully validated the token.
2020-11-19 22:42:18.561 +01:00 [INF] Authorization was successful.
2020-11-19 22:42:18.563 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients)'
2020-11-19 22:42:18.596 +01:00 [INF] Route matched with {action = "PostAdd", controller = "Patients"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] PostAdd(DP.Patients.Controllers.Model.Patient) on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:42:18.660 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients) - Validation state: "Valid"
2020-11-19 22:42:18.689 +01:00 [INF] Entity Framework Core 3.1.8 initialized 'DpDataContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2020-11-19 22:42:18.918 +01:00 [INF] Executed DbCommand (63ms) [Parameters=[@p0='?' (DbType = Int32), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (DbType = DateTime2)], CommandType='"Text"', CommandTimeout='30']
SET NOCOUNT ON;
INSERT INTO [Patients] ([Age], [Email], [Name], [Surname], [TestDate])
VALUES (@p0, @p1, @p2, @p3, @p4);
SELECT [Id]
FROM [Patients]
WHERE @@ROWCOUNT = 1 AND [Id] = scope_identity();
2020-11-19 22:42:19.801 +01:00 [INF] Executed action method DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients), returned result Microsoft.AspNetCore.Mvc.CreatedResult in 1138.026ms.
2020-11-19 22:42:19.807 +01:00 [INF] Executing ObjectResult, writing value of type 'DP.Patients.Controllers.Model.Patient'.
2020-11-19 22:42:19.811 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients) in 1211.3374ms
2020-11-19 22:42:19.816 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients)'
2020-11-19 22:42:19.820 +01:00 [INF] HTTP POST /api/Patients responded 201 in 1931.8193 ms
2020-11-19 22:42:19.824 +01:00 [INF] Request finished in 1940.3744ms 201 application/json; charset=utf-8
2020-11-19 22:43:57.247 +01:00 [INF] Request starting HTTP/1.1 POST https://localhost:5001/api/Patients application/json; charset=utf-8 104
2020-11-19 22:43:57.255 +01:00 [INF] Successfully validated the token.
2020-11-19 22:43:57.282 +01:00 [INF] Authorization was successful.
2020-11-19 22:43:57.286 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients)'
2020-11-19 22:43:57.297 +01:00 [INF] Route matched with {action = "PostAdd", controller = "Patients"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] PostAdd(DP.Patients.Controllers.Model.Patient) on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:43:57.328 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients) - Validation state: "Valid"
2020-11-19 22:43:57.337 +01:00 [INF] Entity Framework Core 3.1.8 initialized 'DpDataContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2020-11-19 22:43:57.444 +01:00 [INF] Executed DbCommand (63ms) [Parameters=[@p0='?' (DbType = Int32), @p1='?' (Size = 4000), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (DbType = DateTime2)], CommandType='"Text"', CommandTimeout='30']
SET NOCOUNT ON;
INSERT INTO [Patients] ([Age], [Email], [Name], [Surname], [TestDate])
VALUES (@p0, @p1, @p2, @p3, @p4);
SELECT [Id]
FROM [Patients]
WHERE @@ROWCOUNT = 1 AND [Id] = scope_identity();
2020-11-19 22:43:57.986 +01:00 [INF] Executed action method DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients), returned result Microsoft.AspNetCore.Mvc.CreatedResult in 654.2539ms.
2020-11-19 22:43:58.016 +01:00 [INF] Executing ObjectResult, writing value of type 'DP.Patients.Controllers.Model.Patient'.
2020-11-19 22:43:58.021 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients) in 719.3175ms
2020-11-19 22:43:58.032 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.PostAdd (DP.Patients)'
2020-11-19 22:43:58.037 +01:00 [INF] HTTP POST /api/Patients responded 201 in 786.7980 ms
2020-11-19 22:43:58.043 +01:00 [INF] Request finished in 796.4961ms 201 application/json; charset=utf-8
2020-11-19 22:44:15.084 +01:00 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/Patients  
2020-11-19 22:44:15.088 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.GetData (DP.Patients)'
2020-11-19 22:44:15.092 +01:00 [INF] Route matched with {action = "GetData", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult GetData() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:44:15.118 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.GetData (DP.Patients) - Validation state: "Valid"
2020-11-19 22:44:15.124 +01:00 [INF] Entity Framework Core 3.1.8 initialized 'DpDataContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2020-11-19 22:44:15.197 +01:00 [INF] Executed DbCommand (68ms) [Parameters=[], CommandType='"Text"', CommandTimeout='30']
SELECT [p].[Id], [p].[Age], [p].[Email], [p].[Name], [p].[Surname], [p].[TestDate]
FROM [Patients] AS [p]
2020-11-19 22:44:15.214 +01:00 [INF] Executed action method DP.Patients.Controllers.PatientsController.GetData (DP.Patients), returned result Microsoft.AspNetCore.Mvc.OkObjectResult in 91.2371ms.
2020-11-19 22:44:15.218 +01:00 [INF] Executing ObjectResult, writing value of type 'System.Collections.Generic.List`1[[DP.Patients.Controllers.Model.Patient, DP.Patients, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]'.
2020-11-19 22:44:15.222 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.GetData (DP.Patients) in 124.7896ms
2020-11-19 22:44:15.226 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.GetData (DP.Patients)'
2020-11-19 22:44:15.229 +01:00 [INF] HTTP GET /api/Patients responded 200 in 142.2836 ms
2020-11-19 22:44:15.233 +01:00 [INF] Request finished in 149.0565ms 200 application/json; charset=utf-8
2020-11-19 22:45:43.517 +01:00 [INF] Request starting HTTP/1.1 POST https://localhost:5001/api/Patients application/json 0
2020-11-19 22:45:43.521 +01:00 [INF] Authorization failed.
2020-11-19 22:45:43.529 +01:00 [INF] AuthenticationScheme: Bearer was challenged.
2020-11-19 22:45:43.531 +01:00 [INF] HTTP POST /api/Patients responded 401 in 10.9829 ms
2020-11-19 22:45:43.536 +01:00 [INF] Request finished in 19.3446ms 401 
2020-11-19 22:46:11.019 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json 0
2020-11-19 22:46:11.024 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:46:11.039 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:46:11.084 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:46:16.236 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 5180.6764ms
2020-11-19 22:46:16.239 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:46:16.242 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:46:16.329 +01:00 [ERR] HTTP PUT /api/Patients responded 500 in 5305.4184 ms
2020-11-19 22:46:16.336 +01:00 [INF] Request finished in 5317.4528ms 500 text/plain
2020-11-19 22:46:52.475 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json 0
2020-11-19 22:46:52.511 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:46:52.518 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:46:52.546 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:46:55.063 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 2538.7942ms
2020-11-19 22:46:55.068 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:46:55.071 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:46:55.091 +01:00 [ERR] HTTP PUT /api/Patients responded 500 in 2581.3324 ms
2020-11-19 22:46:55.101 +01:00 [INF] Request finished in 2626.8326ms 500 text/plain
2020-11-19 22:47:17.064 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json; charset=utf-8 0
2020-11-19 22:47:17.069 +01:00 [INF] Successfully validated the token.
2020-11-19 22:47:17.071 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:47:17.074 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:47:17.099 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:47:19.107 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 2028.2058ms
2020-11-19 22:47:19.110 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:47:19.115 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:47:19.129 +01:00 [ERR] HTTP PUT /api/Patients responded 500 in 2061.9572 ms
2020-11-19 22:47:19.134 +01:00 [INF] Request finished in 2070.2425ms 500 text/plain
2020-11-19 22:47:44.233 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json; charset=utf-8 0
2020-11-19 22:47:44.238 +01:00 [INF] Successfully validated the token.
2020-11-19 22:47:44.242 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:47:44.248 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:47:44.272 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:47:48.838 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 4585.7276ms
2020-11-19 22:47:48.842 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:47:48.855 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:47:48.868 +01:00 [ERR] HTTP PUT /api/Patients responded 500 in 4631.7503 ms
2020-11-19 22:47:48.872 +01:00 [INF] Request finished in 4639.512ms 500 text/plain
2020-11-19 22:47:57.620 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json 0
2020-11-19 22:47:57.638 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:47:57.642 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:47:57.675 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:47:57.736 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 84.8385ms
2020-11-19 22:47:57.739 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:47:57.742 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:47:57.758 +01:00 [ERR] HTTP PUT /api/Patients responded 500 in 119.8955 ms
2020-11-19 22:47:57.762 +01:00 [INF] Request finished in 142.5573ms 500 text/plain
2020-11-19 22:54:17.160 +01:00 [INF] User profile is available. Using 'C:\Users\Bartek\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-11-19 22:54:18.486 +01:00 [INF] Now listening on: https://localhost:5001
2020-11-19 22:54:18.513 +01:00 [INF] Application started. Press Ctrl+C to shut down.
2020-11-19 22:54:18.516 +01:00 [INF] Hosting environment: Development
2020-11-19 22:54:18.527 +01:00 [INF] Content root path: C:\Users\Bartek\source\repos\DP.Patients\DP.Patients
2020-11-19 22:55:31.525 +01:00 [INF] Request starting HTTP/1.1 POST https://localhost:5001/api/Patients application/json 0
2020-11-19 22:55:31.639 +01:00 [INF] Authorization failed.
2020-11-19 22:55:31.650 +01:00 [INF] AuthenticationScheme: Bearer was challenged.
2020-11-19 22:55:31.654 +01:00 [INF] HTTP POST /api/Patients responded 401 in 111.4420 ms
2020-11-19 22:55:31.673 +01:00 [INF] Request finished in 173.4941ms 401 
2020-11-19 22:55:57.162 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json 0
2020-11-19 22:55:57.167 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:55:57.196 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:55:57.425 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:55:57.514 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 310.349ms
2020-11-19 22:55:57.517 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:55:57.520 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:55:57.608 +01:00 [ERR] HTTP PUT /api/Patients responded 500 in 443.7211 ms
2020-11-19 22:55:57.614 +01:00 [INF] Request finished in 452.4644ms 500 text/plain
2020-11-19 22:56:12.948 +01:00 [INF] Request starting HTTP/1.1 PUT https://localhost:5001/api/Patients application/json; charset=utf-8 0
2020-11-19 22:56:13.499 +01:00 [INF] Successfully validated the token.
2020-11-19 22:56:13.505 +01:00 [INF] Executing endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:56:13.508 +01:00 [INF] Route matched with {action = "InvalidAction", controller = "Patients"}. Executing controller action with signature Microsoft.AspNetCore.Mvc.IActionResult InvalidAction() on controller DP.Patients.Controllers.PatientsController (DP.Patients).
2020-11-19 22:56:13.547 +01:00 [INF] Executing action method DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) - Validation state: "Valid"
2020-11-19 22:56:13.623 +01:00 [INF] Executed action DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients) in 109.4328ms
2020-11-19 22:56:13.628 +01:00 [INF] Executed endpoint 'DP.Patients.Controllers.PatientsController.InvalidAction (DP.Patients)'
2020-11-19 22:56:13.632 +01:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Testowy wyjatek
   at DP.Patients.Controllers.PatientsController.InvalidAction() in C:\Users\Bartek\source\repos\DP.Patients\DP.Patients\Controllers\PatientsController.cs:line 50
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-19 22:56:13.671 +01:00 [ERR] HTTP PUT /api/Patients responded 500 in 718.6938 ms
2020-11-19 22:56:13.675 +01:00 [INF] Request finished in 726.5592ms 500 text/plain
